lib/Config/Database.php

```diff
@@ -84,6 +84,7 @@ function defineTables() {
       define('MP_STATISTICS_OPENS_TABLE', Env::$db_prefix . 'statistics_opens');
       define('MP_STATISTICS_UNSUBSCRIBES_TABLE', Env::$db_prefix . 'statistics_unsubscribes');
       define('MP_STATISTICS_FORMS_TABLE', Env::$db_prefix . 'statistics_forms');
+      define('MP_STATISTICS_WOOCOMMERCE_ORDERS_TABLE', Env::$db_prefix . 'statistics_woocommerce_orders');
```

Comment:
Wouldn't be `statistics_woocommerce_purchases` better name? we will not track orders here, we will only track if someone will make a purchase. But this is really just a nitpick, you don't need to change the name now.